executors/cmdexec/device.go

```diff
@@ -80,10 +80,13 @@ func (s *CmdExecutor) DeviceTeardown(host, device, vgid string) error {
 	// Execute command
 	_, err := s.RemoteExecutor.RemoteCommandExecute(host, commands, 5)
 	if err != nil {
-		logger.LogError("Error while deleting device %v with id %v on host %v: %v",
+		return logger.LogError(
```

Comment:
What does `logger.LogError()` return? Should this not be an instance of `error` instead?

Reply:
Ok, so this wrapper does not return the success/fail of the logging, but it returns the formatted string. Who would have thought that!?